configure

```diff
@@ -249,6 +249,16 @@ echo_shell_vars() {
   fi
   echo "PREFIX=$FLAG_prefix"
   echo "DATAROOTDIR=$FLAG_datarootdir"
+  cat >$TMP/ld_flags.c <<EOF
```

Comment:
how about using `build/detect-cc.c` in place of this tmp file?  

(There are a lot of $TMP files I want to get rid of, which are mainly for the CPython build, which will eventually be obsolete)

Reply:
Yeah, that works since I don't care about the contents of the file being compiled. Updated.